iree/turbine/kernel/boo/ops/conv.py

```diff
@@ -482,8 +446,12 @@ def boo_conv(
     """
 
     num_spatial_dims = len(weight.shape) - 2
-
-    _infer = lambda layout: shared_layout or layout or DEFAULT_LAYOUTS[num_spatial_dims]
+    no_layouts = (
+        shared_layout is None
+        and input_layout is None
+        and kernel_layout is None
+        and output_layout is None
+    )
```

Comment:
you could do `no_layouts = all(x is None for x in [shared_layout, input_layout, kernel_layout, output_layout])`